radicle-node/src/service.rs

```diff
@@ -1217,7 +1224,9 @@ mod gossip {
             timestamp,
             alias,
             addresses,
+            nonce: 0,
         }
+        .solve()
```

Comment:
Would a more elaborate name be better?  Something related to proof of work like solve_proof_of_work(..) or gen_proof_of_work(..).  It better clarifies whats being solved.